### DISCUSSION TITLE
Cutting right through a board

### DISCUSSION THREAD

**Discussion by HannahPrinsloo** [arkypita/LaserGRBL#1869] [Q&A]
I am very new to this, i have the engraving part under the belt, but i cannot find any help online wrt cutting right through a 3mm board. I know my laser is able to do that, but I can’t figure out how to set the program to cut through. Please help!

**Comment by arkypita** [comment]
Cutting or engraving is just a question of power and speed.

If the engraving is normally done with power 30-50% and speed of about 1000-2000 mm/min the cut should be done with power of 80-90% and speed 200-500 mm/min are used. Of course you should focus the dot at the smallest point, because if out-of-foucus it does not cut.

You should prepare your image as a silhouette of the contour you want to cut and use "vectorize" option in lasergrbl, or create the cut path in SVG using inkscape.

**Reply by HannahPrinsloo** [reply]
Any chance you could send me step by step photos of the settings?
